### DISCUSSION TITLE
Draggable MultiSelect

### DISCUSSION THREAD

**Discussion by milind** [mantinedev/mantine#1583] [Feature requests]
Hey! Have been using Mantine components briefly through a wrapper of the package ported over to Python and have been loving it! One comment that I had was that it would be nice to allow the multiselect values to be "dragabble" once selected, as the component yields a list (ordered). Right now the best way to order selections is to just choose them in order or "x" out of an option and click it again in order for it to move to the end of the list. I'm sure you have a lot on your plate, but this is just a minor suggestion if you ever get the time. Thanks!

**Comment by rtivital** [comment]
It's not actually minor, it's a pretty big feature to develop and maintain. Furthermore, it will produce some UX issues (how would you add keyboard support for that, for example? What about screen reader support?). So, I doubt that we should add this feature as part of the library.

**Reply by milind** [reply]
Completely understandable - by "minor", I meant from a user interaction perspective, so if the work to implement and maintain that is disproportionate to the benefit, then no reason to address it. (Thanks for all of your great work on this package!)
